### DISCUSSION TITLE
无法登录

### DISCUSSION THREAD

**Discussion by ZZZ-zero** [Mrs4s/go-cqhttp#1746] [Q&A]
今天原来的号被风控，然后在在配置文件里换号，由于扫码登录一直提示网络环境复杂，所以改用网页验证，但是一直提示账密错误或者非常用设备，更换了多个账号都是，并且确认了账密正确![mmexport1666915514284.png](https://user-images.githubusercontent.com/30883525/198421454-62ec9e07-1da5-4776-be5c-f3e5707ee175.png)

**Comment by Smxzrz** [comment]
我今天也发现了这个问题

**Reply by ironfox24** [reply]
我也是
